test/src/test/integration/librad/scenario/tracked_references.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+use crate::{
+    logging,
+    rad::{identities::TestProject, testnet},
+};
+use librad::{
+    git::{
+        identities,
+        storage::ReadOnlyStorage,
+        tracking,
+        types::{Namespace, Reference},
+        util::quick_commit,
+    },
+    git_ext::tree,
+    reflike,
+};
+use std::ops::Index as _;
```

Comment:
Minor nit, some consistency in import order would make my inner OCdaemon happy.

Reply:
Interesting, what consistency are you referring to? `rust-analyzer` arranged these imports in alphabetical order, are you thinking of something like nested imports appearing below non-nested imports?

Reply:
Dropping an example which show-cases order: std, thrid-party crates, first-party crates, super, crate:

https://github.com/radicle-dev/radicle-link/blob/33c0db4402fb81c1e10bd7742531d972afc98379/librad/src/net/protocol.rs#L6-L33

Reply:
Thanks! Interestingly you have to have a new line between each category or `cargo fmt` will just silently reorder everything to alphabetical order for you.